README.md

```diff
@@ -6,15 +6,26 @@ Goal is to use 100% (or close to it) Elm on the front-end and Elixir w/ [Phoenix
 
 ## Development
 
-To install prerequisites run:
+Prerequisites
+  - Elixir v1.3
 
-    $ mix deps.get
-    $ npm install
+Install Elm:
+
+    $ cd web/elm
+    $ elm package install
```

Comment:
This Install Elm section can be part of the To install dependencies run section. Just put it after the $npm install line